src/widget/wstarrating.h

```diff
@@ -32,7 +32,9 @@ class WStarRating : public QWidget, public WBaseWidget {
     void mouseMoveEvent(QMouseEvent *event);
     void mouseReleaseEvent(QMouseEvent *event);
     void leaveEvent(QEvent *);
-    
+    virtual bool event(QEvent* pEvent);
+    void fillDebugTooltip(QStringList* debug);
```

Comment:
The master branch is already on C++11. All "override" comments have been 
replaced by the keyword.

On 09/15/2015 02:02 AM, Jean Claveau wrote:
>
> In src/widget/wstarrating.h 
> <https://github.com/mixxxdj/mixxx/pull/685#discussion_r39462020>:
>
> > @@ -32,7 +32,9 @@ class WStarRating : public QWidget, public WBaseWidget {
> >      void mouseMoveEvent(QMouseEvent *event);
> >      void mouseReleaseEvent(QMouseEvent *event);
> >      void leaveEvent(QEvent *);
> > -
> > +    virtual bool event(QEvent* pEvent);
> > +    void fillDebugTooltip(QStringList* debug);
>
> It is done. I added "virtual" and "/override/" as comment even if I've not 
> been able to find this string somewhere else in the sources. Please, tell 
> me if I'm wrong.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/mixxxdj/mixxx/pull/685/files#r39462020>.
>